### PR TITLE
update Creative Commons licenses to 4.0. fixes #1286

### LINKED ISSUE #1286: Support Creative Commons 4.0
I noticed that DokuWiki only supports Creative Commons version 3.0. I would rather use version 4.0.

### PR DESCRIPTION
**Pull request by polyzen**
https://wiki.creativecommons.org/wiki/License_versions
https://wiki.creativecommons.org/wiki/Frequently_Asked_Questions#Why_should_I_use_the_latest_version_of_the_Creative_Commons_licenses.3F

### CODE REVIEW

File: conf/license.php [hunk 1]
```diff
@@ -14,23 +14,23 @@
     'url'  => 'http://creativecommons.org/licenses/publicdomain/',
 );
 $license['cc-by'] = array(
-    'name' => 'CC Attribution 3.0 Unported',
-    'url'  => 'http://creativecommons.org/licenses/by/3.0/',
+    'name' => 'CC Attribution 4.0 International',
```
Comment (Klap-in) [inline conf/license.php]: Is there a difference between 'Unported' and 'International'?

Comment (polyzen) [inline conf/license.php]: [Apparently not](https://wiki.creativecommons.org/wiki/Frequently_Asked_Questions#What_are_the_international_.28.E2.80.9Cunported.E2.80.9D.29_Creative_Commons_licenses.2C_and_why_does_CC_offer_.E2.80.9Cported.E2.80.9D_licenses.3F)

Comment (Klap-in) [inline conf/license.php]: Thanks, that is a clear answer.

**Comment by Klap-in** [comment]
:+1:

**Comment by Klap-in** [comment]
Hey @polyzen, why are you closing this?

**Comment by polyzen** [comment]
Sorry about that, @Klap-in :p I forgot this hasn't been merged yet

**Pull request merged**